modules/gitops-accessing-gitops-monitoring-dashboards.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/monitoring-with-gitops-dashboards.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-accessing-gitops-monitoring-dashboards_{context}"]
+= Accessing {gitops-shortname} monitoring dashboards
+
+The monitoring dashboards are deployed automatically by the Operator. You can access {gitops-shortname} monitoring dashboards from the *Administrator* perspective of the {OCP} web console. 
+
+[NOTE]
+====
+Disabling or changing the content of the dashboards is not supported.
+====
+
+.Prerequisites
+
+* You have access to the {OCP} web console.
+* The {gitops-title} Operator is installed in the default namespace, `openshift-gitops-operator`.
+* The cluster monitoring is enabled on the `openshift-gitops-operator` namespace.
```

Comment:
@ekristova Did you test this step and option with the latest 1.10 GitOps  Operator in the cluster?

Reply:
@Srivaralakshmi I suppose you mean the step to enable monitoring, yes I tested it and the whole procedure :)

Reply:
Here is my screenshot:
![image](https://github.com/openshift/openshift-docs/assets/98941675/f06e5ad0-fe71-4343-ac83-59f0c619335f)